README.md

```diff
@@ -39,15 +39,15 @@ Meltano Airflow utility extension
       value: false
     - name: database.sql_alchemy_conn
       label: SQL Alchemy Connection
-      value: sqlite:///$MELTANO_PROJECT_ROOT/airflow/airflow.db
+      value: sqlite:///$MELTANO_PROJECT_ROOT/orchestrate/airflow/config/airflow.db
```

Comment:
Did we revert this decision re the config subfolder?

Reply:
No ? This is the subfolder structured you referenced in https://github.com/meltano/hub/pull/822#issuecomment-1276804146:

```
├── meltano.yml
├── orchestrate
│   └── airflow
│       ├── .gitignore
│       ├── dags
│           └── ...
│       ├── logs
│           └── ...
│       ├── plugins
│           └── ...
│       ├── config (or home)
│           └── README.md
│           └── airflow.db
│           └── airflow.cfg
│           └── webserver_config.py
```

The only difference being that webserver_config.py will end up in AIRFLOW_HOME (i.e. `PROJECT_ROOT/orchestrate/airflow`)

Reply:
Thanks, @pandemicsyn. Just checking. I still like the idea of a `config` subfolder but I guess I thought the constraint on `webserver_config.py` being tied to the `AIRFLOW_HOME` dir also extended to the other config files.
  
Pardon my ignorance on this front. I do like the `config` subfolder being used here; no objection on my part. 👍